### DISCUSSION TITLE
User does not have any Google Analytics account

### DISCUSSION THREAD

**Discussion by BaraaDev** [freekmurze/laravel-analytics#431] [Q&A]
{ "error": { "code": 403, "message": "User does not have sufficient permissions for this profile.", "errors": [ { "message": "User does not have sufficient permissions for this profile.", "domain": "global", "reason": "insufficientPermissions" } ] } }

**Comment by abishekrsrikaanth** [comment]
@freekmurze any idea what would cause this error. Just started using this.

**Reply by salmanaugmentler** [reply]
You can refer to this [Comment](https://github.com/spatie/laravel-analytics/discussions/374#discussioncomment-4138086) Assuming that you have Universal ID enabled as per Laravel Analytics docs.
